libcxx/include/string_view

```diff
@@ -686,6 +686,12 @@ template <class _CharT, class _Traits>
 inline constexpr bool ranges::enable_borrowed_range<basic_string_view<_CharT, _Traits> > = true;
 #endif // _LIBCPP_STD_VER >= 20
 
+template <class _CharT, class _Traits>
+struct __tombstone_memory_layout<basic_string_view<_CharT, _Traits>> {
+  static constexpr size_t __disengaged_value_     = basic_string_view<_CharT, _Traits>::npos;
```

Comment:
I think `npos` is a valid size for a `string_view`. It's somewhat academic but I don't think this specific optimization is valid.